python/docs/examples.rst

```diff
@@ -1,8 +1,205 @@
 Examples
-========
+~~~~~~~~
+
+Connecting to Feldera Sandbox
+=============================
+
+.. code-block:: python
+
+    from feldera import FelderaClient, PipelineBuilder
+
+    client = FelderaClient('https://try.feldera.com', api_key=api_key)
+
+    pipeline = PipelineBuilder(client, name, sql).create()
+
+Connecting to Feldera on localhost
+==================================
+
+.. code-block:: python
+
+    from feldera import FelderaClient, PipelineBuilder
+
+    client = FelderaClient('https://try.feldera.com', api_key=api_key)
+
+    pipeline = PipelineBuilder(client, name, sql).create()
+
+Creating a Pipeline
```

Comment:
```suggestion
Creating a Pipeline (OVERWRITING existing pipeline)
```

Since people may copy-paste this code, we should emphasize that this will shutdown and overwrite any existing pipeline with this name.